### PR TITLE
2021 submission stats

### PR DESCRIPTION
**Pull request by mhl10**

### CODE REVIEW

File: help/stats/2021_by_area/index.html [hunk 1]
```diff
@@ -0,0 +1,358 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (mhl10) [inline help/stats/2021_by_area/index.html:2]: This is the file originally supplied by PG; it has been translated to markdown with index.html.

**Review by afromme — APPROVED** [review]
Looks great, thanks!

File: help/stats/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ arXiv usage statistics
 
 -   [Monthly submission totals](https://arxiv.org/stats/monthly_submissions)
 -   [Submissions by Category](/about/reports/submission_category_by_year)
--   [Submission totals broken down by major subject areas through 2020](2020_by_area/index)
+-   [Submission totals broken down by major subject areas through 2021](2021_by_area/index)
```
Comment (jimentwood) [inline help/stats/index.md:8]:
```suggestion
-   [Submission totals broken down by major subject areas through 2021](2021_by_area)

```

suggest dropping the 'index' portion of the url

**Review by bmaltzan — APPROVED** [review]
Looks good!

**Review by jimentwood — APPROVED** [review]
looks good - made one trivial suggestion

**Comment by SteinnSigurdsson** [comment]
nice

**Pull request merged**